### PR TITLE
Adapt to Starcounter/starcounter-include@3.0.0,

### PR DESCRIPTION
**Pull request by tomalec**
add explicit slot names and elements.
fixes https://github.com/Starcounter/starcounter-layout-html-editor/pull/14#issuecomment-319936761

### CODE REVIEW

**Comment by warpech** [comment]
@PatrykSzwer can you pls review?

File: src/Products/wwwroot/Products/viewmodels/ProductTypesPage.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <link rel="import" href="/sys/x-breadcrumb/x-crumb.html" />
 
 <style type="text/css">
-    
+
 .kitchensink-x-breadcrumb {
```
Comment (PatrykSzwer) [inline src/Products/wwwroot/Products/viewmodels/ProductTypesPage.html]: **kitchensink**-x-breadcrumb? Is it just leftover from coping classes from kitchensink or this have some sense? If the first one could You please change it?

Comment (tomalec) [inline src/Products/wwwroot/Products/viewmodels/ProductTypesPage.html]: Seems like a leftover after some coping, I was not touching actual CSS.  Will fix that. thanks for noticing.

**Comment by tomalec** [comment]
I have resolved the conflicts. But I think It'd be cleaner to cover CSS fixes in separate PR https://github.com/StarcounterApps/Products/issues/60 as it is slightly separated issue.

**Pull request merged**